tests/app/template/test_rest.py

```diff
@@ -238,13 +242,14 @@ def test_should_not_create_template_with_incorrect_provider_type(
     assert json_resp['message'] == f'invalid {template_type}_provider_id'
 
 
-@pytest.mark.xfail(reason='Mislabelled for route removal, fails when unskipped', run=False)
+@pytest.mark.skip(reason='TODO #2336 - Fail due to orphaned User object')
+@pytest.mark.serial
```

Comment:
ditto not serial

Reply:
Updated and remove skip.

Confirm test passes locally and on Github Jobs.